### DISCUSSION TITLE
How to prevent nude images generated perfectly?

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#2941] [Q&A] [answered]
I used the negative prompt as " Two-piece, Bikini briefs, Monokini, Tankini, Triangle bikini, Bandeau bikini,Halter-neck bikini, High-waisted bikini, naked,naked, bachelorette, underwearing, underweared, nuke, nudity, bachelor, bottomless, underwear, bikini ,  bikini ,  bikini ,  bikini ,  bikini ,  bikini , topless,underwearing, underweared,underwearing, underweared, sexy, around current clothing,"

But sometimes it shows nude pictures.

Please let me know What else I should do.

**Comment by ILikeToasters** [comment, marked as answer]
See my response to your other post about the same thing.

**Comment by mashb1t** [comment]
@oldhand7 can you please stop spamming discussions multiple times, if you don't receive an answer a few hours later?
Discussions will be read and will also be answered, if the question is not complete nonsense.
Last warning before formal report to Github, you were already warned. Thank you for understanding. 

reference: https://github.com/lllyasviel/Fooocus/discussions/2908#discussioncomment-9417944
